fdbclient/include/fdbclient/ServerKnobs.h

```diff
@@ -228,7 +228,6 @@ class SWIFT_CXX_IMMORTAL_SINGLETON_TYPE ServerKnobs : public KnobsImpl<ServerKno
 	bool ENABLE_DD_PHYSICAL_SHARD; // EXPERIMENTAL; If true, SHARD_ENCODE_LOCATION_METADATA must be true.
 	double DD_PHYSICAL_SHARD_MOVE_PROBABILITY; // Percentage of physical shard move, in the range of [0, 1].
 	bool ENABLE_PHYSICAL_SHARD_MOVE_EXPERIMENT;
-	bool BULKLOAD_ONLY_USE_PHYSICAL_SHARD_MOVE; // If true, bulk load only uses physical shard move
```

Comment:
We are removing this flag because we will auto-detect whether to do physical shard move or not? Or is it that we'll restore physical shard move under a knob at a later date?

Reply:
Currently, we have two data move types: (1) fetchKey based data moves (for general storage engine); (2) physical shard move (experimental at current stage and specific to sharded rocksdb). This knob is used to enforce to use physical shard move for bulkload. This PR allows bulkload to run with fetchKey based data move. So, this knob is no longer needed.